.github/workflows/build-libraries.yml

```diff
@@ -98,4 +96,5 @@ jobs:
         run: |

```

Comment:
What do you think about using `ctest.exe -VV build`?

Reply:
Replaced with more verbose keys